modules/p2/ding_list/plugins/content_types/private_share.inc

```diff
@@ -51,8 +51,10 @@ function ding_list_private_share_content_type_render($subtype, $conf, $panel_arg
           'attributes' => array(
             'class' => array(
               'sharer-button', 'sharer-' . $community, 'use-ajax',
+              'sharer-noshare',
```

Comment:
As I read it this is adding the `sharer-noshare` class to elements that also have an `sharer-facebook` class. That seems odd - perhaps this warrants some refactoring?